### ISSUE TITLE
Triangle generated from simplification face in opposite direction from input.

### ISSUE THREAD

**Issue by RogersDoug** [valette/ACVD#17]
In ACVDQ, the winding order from the triangles in the simplified mesh appears to be backwards.

**Comment by valette** [comment]
Do you have a small example for that? The code is supposed to output meshes orientated consistently with the input mesh.

**Comment by RogersDoug** [comment]
You can see the culling is incorrect in the window.  I added the following code in RenderWindow.cxx to enable back face culling.  Hitting 'c' when the window is open toggles backface culling.

This model should show the issue:

https://www.dropbox.com/s/je2eor0c1dwt4ec/horse.ply?dl=0

```
	virtual void OnChar ()
	{
		vtkRenderWindowInteractor *rwi = this->Interactor;
		switch (rwi->GetKeyCode())
		{

		case 'c':
		{
			bool culling =  this->Window->GetMeshActor()->GetProperty()->GetBackfaceCulling();

			this->Window->GetMeshActor()->GetProperty()->SetBackfaceCulling(!culling);
		}

```

**Comment by valette** [comment]
This patch should fix it...

**Comment by valette** [comment]
I have also included your patch for backface culling as e684c0ed7567b4b327071d6138be2f34c93b5850.

Feel free to reopen the issue if the bug persists...
